build.gradle

```diff
@@ -41,6 +41,7 @@ project.ext.externalDependency = [
     'commonsLang': 'commons-lang:commons-lang:2.6',
     'commonsCollections': 'commons-collections:commons-collections:3.2.2',
     'data' : 'com.linkedin.pegasus:data:' + pegasusVersion,
+    'dgraph4j' : 'io.dgraph:dgraph4j:21.03.1',
```

Comment:
We have different DAOs in [datahub-gma](https://github.com/linkedin/datahub-gma), for example, gmaNeo4jDao. How about adding DgraphDAO?

Reply:
What would the purpose be? Who would be using it other than the `DgraphGraphService`?